indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -32,8 +32,8 @@
 from indico.modules.events.abstracts.models.review_questions import AbstractReviewQuestion
 from indico.modules.events.abstracts.models.review_ratings import AbstractReviewRating
 from indico.modules.events.models.persons import AuthorsSpeakersMixin
+from indico.modules.events.models.reviews import ProposalMixin
 from indico.modules.events.contributions.models.contributions import _get_next_friendly_id, CustomFieldsMixin
```

Comment:
I think the import was at least partially unused at the point of writing this comment.

However, `_get_next_friendly_id` should probably be renamed to `get_next_friendly_contrib_id` since it's used outside its own module.